content/10.diagnostics.md

```diff
@@ -81,11 +91,18 @@ The authors are now working to get the assay into clinical use [@url:https://www
 
 Importantly, false-positives can occur due to the cross-reactivity with other antibodies according to the clinical condition of the patient [@url:https://www.fda.gov/media/136625/download].
 Therefore, this test should be used in combination with RNA detection tests [@url:https://www.fda.gov/media/136625/download].
+Serological tests also require waiting until a patient has had an imune system reaction sufficient to detect, which means that these tests are of limited use for diagnosis in the early stages of an infection.
+The limitation on timing makes serological tests vfar less useful for enabling test-and-trace strategies.
 
 ### Possible Alternatives to Current Practices for Identifying Active Cases
 
-- Are there other approaches that have worked for diagnosing other viruses at a rapid pace in large numbers of people?
-- What are some approaches people are currently testing for detecting live viruses, especially SARS-CoV-2?
+Clinical symptoms are similar enough to other types of pneumonia that purely symptomatic diagnosis is unlikely to be sufficient.
```

Comment:
"so that" would clarify that "that" doesn't refer to the pneumonia but to the sentence.

Reply:
Per @matfax comment, I actually don't mind the that there, but I would suggest the following change to the first part of the sentence:
```suggestion
Clinical symptoms are sufficiently similar to other types of pneumonia that purely symptomatic diagnosis is unlikely to be sufficient.
```

Reply:
Sufficiently similar to be sufficient? I don't know...
Clinical symptoms are too similar to other types of pneumonia to be sufficient/reliable as a sole/single diagnostics criterion?